extension/oidcextension/testdata/config.yaml

```diff
@@ -0,0 +1,26 @@
+extensions:
```

Comment:
Do you have receiver test cases for grpc configuration & http configuration? If not lets add them

Reply:
Not at the moment. The current issue there is that the receiver needs to support the server config options which the `examplereceiver` doesn't. So I would need to stand up a full e2e test with a receiver, a grpc client, and the test oidc server to even get anywhere.